content/recipes/swc.md

```diff
@@ -95,3 +95,90 @@ If your ORM does not provide a similar workaround, you can define the wrapper ty
  */
 export type WrapperType<T> = T; // WrapperType === Relation
 ```
+### Vitest
+
+[Vitest](https://vitest.dev/) is a fast and lightweight test runner designed to work with Vite. It provides a modern, fast, and easy-to-use testing solution that can be integrated with NestJS projects.
+
+#### Installation
+
+To get started, first install the required packages:
+
+```bash
+$ npm i --save-dev vitest unplugin-swc @swc/core @vitest/coverage-c8
+```
+
+#### Configuration
+
+Create a vitest.config.ts file in the root directory of your application with the following contents:
+
+```ts
+import swc from 'unplugin-swc';
+import { defineConfig } from 'vite';
```

Comment:
Hi! I was trying to migrate jest to vitest. This will help me!

I wonder if we can use `vitest` here instead of `vite`